catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/taxonomy/ContactAttributes.java

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.impl.taxonomy;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.taxonomy.Contact;
+
+/**
+ *  This class provides attributes that reflect metadata about different kinds of
+ *  people/groups/units/organizations associated with the metacard. This consists
+ *  of the creator, contributor, publisher and the point of contact.
+ *
+ *  https://codice.atlassian.net/wiki/display/DDF/Attributes
+ */
+public class ContactAttributes implements Contact, MetacardType {
```

Comment:
I am not sure this class and similar classes should implement MetacardType.  This shouldn't be a MetacardType unto itself.  It is just a composition of attributes in a category.  Perhaps we need a new interface here?